css/menu-styles.css

```diff
@@ -0,0 +1,64 @@
+.paw-logo {
+  display: inline-block;
+}
+.menu-grid {
+  max-width: 1072px;
+  margin: 200px auto 100px;
+  background-color: #000000;
+  border-radius: 20px;
+  border: 12px solid #ffc0cb;
+  box-sizing: border-box;
+  padding: 60px;
+}
+.menu-grid__heading-title {
+  font-size: 128px;
+  color: #ffc0cb;
+  margin-top: 0;
+  margin-bottom: 80px;
+}
+.menu-grid__container {
+  display: grid;
+  grid-template-columns: repeat(2, 440px);
+  grid-template-rows: repeat(5, 300px);
+  gap: 60px;
+}
+.menu-grid__item {
+  position: relative;
+  border: 12px solid #ffc0cb;
```

Comment:
Girl, but can we use the helper class from the home-page and over write it with 12 px?

Reply:
already discussed in Discord "if I'm not mistaken, I used helper classes from main in all cases except this one. initially I used it and then decided to write it's own property as it has different border width and has no shadow, so instead of writing 2 lines of code I wrote one))))...maybe it's not correct, but it was my vision at that moment😄"